aten/src/ATen/cuda/CUDAContext.h

```diff
@@ -10,6 +10,10 @@
 #include <cusolverDn.h>
```

Comment:
Should this file be hipified to hipsolver/hipsolver.h?

Reply:
@pruthvistony it is. CUDAContext is the pre-hipified file. After hipification the file is located at aten/src/ATen/hip/HIPContext.h. I confirmed hipsolver/hipsolver.h is included from this file.